### DISCUSSION TITLE
是否会在后续添加保留浮窗位置

### DISCUSSION THREAD

**Discussion by baiyao105** [Class-Widgets/Class-Widgets#172] [Q&A问答] [answered]
班级电脑频繁开关,每次都需要拖动,我认为不太方便.

**Comment by baiyao105** [comment]
在最新构建的b4版本中均缺失了提示声,记得加( ;win11桌面状态下误判为全屏收起

**Reply by RinLit-233-shiroko** [reply]
> 在最新构建的b4版本中均缺失了提示声,记得加( ;win11桌面状态下误判为全屏收起

是构建的问题吧，日志显示为：

> IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!
> 
> Importing the numpy C-extensions failed. This error can happen for
> many reasons, often due to issues with your setup or how NumPy was
> installed.
> 
> We have compiled some common reasons and troubleshooting tips at:
> 
>     https://numpy.org/devdocs/user/troubleshooting-importerror.html
> 
> Please note and check the following:
> 
>   * The Python version is: Python3.8 from "X:\ClassWidgets\ClassWidgets.exe"
>   * The NumPy version is: "1.24.4"
> 
> and make sure that they are the versions you expect.
> Please carefully study the documentation linked above for further help.
> 
> Original error was: DLL load failed while importing _multiarray_umath: The specified module could not be found.

到时候我看看咋搞

**Reply by baiyao105** [reply]
👌

**Comment by baiyao105** [comment, marked as answer]
闲的没事干给以前问题解决了(, #443 #444
